etna/transforms/decomposition/change_points_segmentation.py

```diff
@@ -0,0 +1,119 @@
+from typing import Optional
+
+import pandas as pd
+from ruptures.base import BaseEstimator
+
+from etna.transforms.base import FutureMixin
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.decomposition.change_points import ChangePointsTransform
+
+
+class _OneSegmentChangePointsSegmentationTransform(ChangePointsTransform):
+    """_OneSegmentChangePointsSegmentationTransform make label encoder to change points."""
+
+    def __init__(
+        self, in_column: str, change_point_model: BaseEstimator, out_column: str, **change_point_model_predict_params
+    ):
+        """Init _OneSegmentChangePointsSegmentationTransform.
+        Parameters
+        ----------
+        in_column:
+            name of column to apply transform to
+        change_point_model:
+            model to get change points
+        out_column: str, optional
+            result column name. If not given use ``self.__repr__()``
+        change_point_model_predict_params:
+            params for ``change_point_model.predict`` method
+        """
+        super(_OneSegmentChangePointsSegmentationTransform, self).__init__(
+            in_column=in_column, change_point_model=change_point_model, **change_point_model_predict_params
+        )
+
+        self.out_column = out_column
+
+    def _fill_per_interval(self, series: pd.Series) -> pd.Series:
+        """Fill values in resulting series."""
+        if self.intervals is None:
+            raise ValueError("Transform is not fitted! Fit the Transform before calling transform method.")
+        result_series = pd.Series(index=series.index)
+        for k, interval in enumerate(self.intervals):
+            tmp_series = series[interval[0] : interval[1]]
+            if tmp_series.empty:
+                continue
+            result_series[tmp_series.index] = k
+        return result_series.astype(int).astype("category")
+
+    def transform(self, df: pd.DataFrame) -> pd.DataFrame:
+        """Split df to intervals.
+
+        Parameters
+        ----------
+        df:
+            one segment dataframe
+        Returns
```

Comment:
does it look good in docs?
I mean no empty line between `Parameters` and `Returns` sections